$NetBSD$

* install man pages to ${PREFIX}/man
* do not generate ps, pdf, and html documents

--- Makefile.orig	2011-01-31 21:06:08.000000000 +0000
+++ Makefile
@@ -62,7 +62,7 @@ datadir		= $(datarootdir)
 docsubdir	= $(PACKAGE)-$(DOS2UNIX_VERSION)
 docdir		= $(datarootdir)/doc/$(docsubdir)
 localedir	= $(datarootdir)/locale
-mandir		= $(datarootdir)/man
+mandir		= $(prefix)/man
 man1dir		= $(mandir)/man1
 manext		= .1
 man1ext		= .1
@@ -77,7 +77,7 @@ ifdef ENABLE_NLS
 	MOFILES			= $(DOS2UNIX_MOFILES) $(UNIX2DOS_MOFILES)
 endif
 
-DOCFILES	= $(PACKAGE).txt $(PACKAGE).ps $(PACKAGE).pdf $(PACKAGE).htm
+DOCFILES	= $(PACKAGE).txt
 INSTALL_OBJS_DOC = README.txt NEWS.txt ChangeLog.txt COPYING.txt TODO.txt BUGS.txt $(DOCFILES)
 
 #PODFILES	= man/man1/dos2unix.pod $(wildcard man/*/man1/dos2unix.pod)
@@ -274,11 +274,11 @@ else
 	cd $(DESTDIR)$(man1dir); $(LINK_MAN) $(PACKAGE).1 $(UNIX2DOS).1
 	cd $(DESTDIR)$(man1dir); $(LINK_MAN) $(PACKAGE).1 $(UNIX2MAC).1
 endif
-	$(foreach manfile, $(MANFILES), $(MKDIR) -p -m 755 $(DESTDIR)$(datarootdir)/$(dir $(manfile)) ;)
-	$(foreach manfile, $(MANFILES), $(INSTALL) -m 644 $(manfile) $(DESTDIR)$(datarootdir)/$(dir $(manfile)) ;)
-	$(foreach manfile, $(MANFILES), cd $(DESTDIR)$(datarootdir)/$(dir $(manfile)) ; $(LINK_MAN) $(PACKAGE).1 $(MAC2UNIX).1 ;)
-	$(foreach manfile, $(MANFILES), cd $(DESTDIR)$(datarootdir)/$(dir $(manfile)) ; $(LINK_MAN) $(PACKAGE).1 $(UNIX2DOS).1 ;)
-	$(foreach manfile, $(MANFILES), cd $(DESTDIR)$(datarootdir)/$(dir $(manfile)) ; $(LINK_MAN) $(PACKAGE).1 $(UNIX2MAC).1 ;)
+	$(foreach manfile, $(MANFILES), $(MKDIR) -p -m 755 $(DESTDIR)$(prefix)/$(dir $(manfile)) ;)
+	$(foreach manfile, $(MANFILES), $(INSTALL) -m 644 $(manfile) $(DESTDIR)$(prefix)/$(dir $(manfile)) ;)
+	$(foreach manfile, $(MANFILES), cd $(DESTDIR)$(prefix)/$(dir $(manfile)) ; $(LINK_MAN) $(PACKAGE).1 $(MAC2UNIX).1 ;)
+	$(foreach manfile, $(MANFILES), cd $(DESTDIR)$(prefix)/$(dir $(manfile)) ; $(LINK_MAN) $(PACKAGE).1 $(UNIX2DOS).1 ;)
+	$(foreach manfile, $(MANFILES), cd $(DESTDIR)$(prefix)/$(dir $(manfile)) ; $(LINK_MAN) $(PACKAGE).1 $(UNIX2MAC).1 ;)
 ifdef ENABLE_NLS
 	@echo "-- install-mo"
 	$(foreach mofile, $(MOFILES), $(MKDIR) -p -m 755 $(DESTDIR)$(localedir)/$(basename $(notdir $(mofile)))/LC_MESSAGES ;)
